### DISCUSSION TITLE
Settings keep disappearing

### DISCUSSION THREAD

**Discussion by BelvedereHenrique** [TEHE-Studios/ExpandedHelicopterEvents#193] [Q&A] [answered]
Hi!,

I've installed the mod and its settings keep disappearing from my server files.
I don't know if it's a G-portal issue or mod issue.

**Comment by BelvedereHenrique** [comment]
I've found this on the logs
WARN : Mod         , 1688339252588> 313,262,516> ZomboidFileSystem.loadModAndRequired> required mod "SwapIt" not found
WARN : Mod         , 1688339252589> 313,262,516> ZomboidFileSystem.loadModAndRequired> required mod "TMC_TrueActions" not found
WARN : Mod         , 1688339252589> 313,262,517> ZomboidFileSystem.loadModAndRequired> required mod "ExpandedHelicopterEvents" not found

**Comment by BelvedereHenrique** [comment]
I've added ExpandedHelicopterEvents on "Mods" and "2458631365" as the ID

**Comment by BelvedereHenrique** [comment]
I managed to fix the other 2 mods by adding pre-required mods.
But this still wont work.

**Comment by BelvedereHenrique** [comment]
Opened issue: https://github.com/TEHE-Studios/ExpandedHelicopterEvents/issues/194

**Reply by nuclearchickenz** [reply]
Hi!

What exactly did you do to fix this? I'm getting this:
WARN : Mod         , 1688922829631> 1,059,039,796> ZomboidFileSystem.loadModAndRequired> required mod "EasyConfigChucked" not found.
WARN : Mod         , 1688922829633> 1,059,039,799> ZomboidFileSystem.loadModAndRequired> required mod "ExpandedHelicopterEvents" not found.

Deleting the mod files in the workshop folder didn't solve the issue. Now server isn't even downloading them. Thanks!

**Reply by BelvedereHenrique** [reply, marked as answer]
The game (or the server, I'm not really sure) holds a cache of what mod it thinks you've downloaded. So, to make it re-download everything, you have to delete not only the mod folder, but the whole steamApps folder.
![image](https://github.com/TEHE-Studios/ExpandedHelicopterEvents/assets/24565006/530fd45b-a0c1-48de-a8cf-ac4c2a2acda9)
This will force a re-download of everything.
